app/src/debug/google-services.json

```diff
@@ -62,23 +62,23 @@
       "client_info": {
         "mobilesdk_app_id": "1:1022635885638:android:3ba5a7d13d995fc2",
         "android_client_info": {
-          "package_name": "com.simplecity.amp_pro"
+          "package_name": "edu.usf.sas.pal.muser"
```

Comment:
Hmm, looks like the app already uses Firebase for something? We'll need to generate a new Google-services.json file for our app - I'll work on that. Could you check the Github Actions build failure to see what the problem is?

Reply:
Also, please chekc the app and see what Firebase Google Services it currently seems to depend on, we'll need to switch those on.

Reply:
I checked the build logs and I think it's a syntax error and also certificate hash was missing from the JSON object. I have pushed a commit correcting the error and restoring the certificate to the request and it seems to have fixed the issue.